test/server/GraknRunnerBase.java

```diff
@@ -182,6 +181,7 @@ private void distributionSetup(Path distributionDir, Path dataDir) throws IOExce
         System.out.println(graknPath);
         executor = executor.directory(graknPath.toFile());
         System.out.println(name() + " distribution unarchived");
+        dataDir = graknPath.resolve("server").resolve("data");
```

Comment:
at this point, the variable is never even used. Why do we even need it now?

Reply:
@haikalpribadi it's used in subclasses, e.g. [here](https://github.com/graknlabs/common/blob/2868372698333b989b429268f03e0a9107bd1e4f/test/server/GraknCoreRunner.java#L59)

Reply:
The value is used in another function, `start()` which does not appear in the diff since it is unchanged

Reply:
right, I see